docs/us.md

```diff
@@ -0,0 +1,225 @@
+# Update system consensus rules
+
+## Prerequisites
+
+* [Update Mechanism](https://cardanodocs.com/cardano/update-mechanism/).
+
+## Overview
+Update system gives ability to developers to propose updates of software applications and
+ability to users to vote for updates to decide which one will be accepted.
+
+This document describes a global part of update system consensus rules, as well difference between a local part.
+Global consensus rules is a part of block processing, which checks validity of update payload of blocks and update 
+corresponding state when blocks are applied/rolled back. 
+We first describe global part and then describe how local part differs.
+
+## Problem definition
+TBA
+
+## Update system model
+
+### Poll and decision agreement rules
+
+Each stakeholder can send update proposal in the blockchain,
+and others stakeholders can send their votes "for" or "against" it.
+Making decision whether approve or reject an update proposal depends on sum of stake "for" and "against" it.
+
+There are two rules to make decision about update proposal:
+* **Explicit agreement**: if an update proposal has greater than 50% stake "for" or "against" 
+then the proposal becomes approved or rejected corresponding.
+* **Implicit agreement**: if an update proposal has been proposed in the blockchain `U` slots ago,
+and it has more stakes "for" then "against", then the proposal becomes approved and rejected otherwise.
+
+### Update proposal states
+
+Update proposal may be in three states:
+* **Active** (or Undecided)  
+When an update proposal gets into the blockchain within some block, it becomes _active_.
+This state means that the proposal is known in the blockchain but 
+it wasn't still decided by stakeholders whether to approve or to reject this proposal.
+So, a poll is active and stakeholders' votes which get into the blockchain affect the decision.
+
+* **Decided**  
+An update proposal continues being _active_ until it becomes _decided_, it may be _decided_ to approve
+or to reject.
+If a proposal is rejected then it doesn't affect any consensus rules and we don't consider it anymore.
+Note: rejected proposal may become active again if rollback occurs.
+
+* **Confirmed**  
```

Comment:
You forgot **Discarded** state. One may argue that proposal can't be in this state, because when it's discarded, it's, well, literally discarded (i. e. thrown away from the consideration), but I think it's still useful to have this notion in some cases.